### PR TITLE
Add recipient ID for Gemini Connections

### PR DESCRIPTION
**Pull request by corymcdonald**
#### Features

- Adds recipient id to the Gemini Connections

#### How To Test

Run `rails test`

~ Alternatively ~

You can connect your Publishers account to Gemini and then check the database by running 
`bundle exec rails console` 
One into the Rails console you can check all the recipient_ids
`GeminiConnection.all.map(&:recipient_id)`

### CODE REVIEW

File: app/models/publisher.rb [hunk 1]
```diff
@@ -160,6 +160,20 @@ def wallet
...
+  # Public: Checks the different wallet connections and enqueues sync jobs to refresh their data
+  #         If their data wasn't refreshed in the last 2 hours.
+  #
+  # Returns nil
+  def sync_wallet_connections
+    if gemini_connection.present? && gemini_connection.updated_at > 2.hours.ago
+      Sync::Connection::GeminiConnectionSyncJob.perform_later(publisher_id: id)
+    end
+
+    if uphold_connection.present? && uphold_connection.updated_at > 2.hours.ago
```
Comment (yachtcaptain23) [inline app/models/publisher.rb:172]:
```suggestion
    if uphold_connection.present? && uphold_connection.updated_at < 2.hours.ago
```

File: app/models/publisher.rb [hunk 2]
```diff
@@ -160,6 +160,20 @@ def wallet
     @wallet ||= PublisherWalletGetter.new(publisher: self, include_transactions: false).perform
   end
 
+  # Public: Checks the different wallet connections and enqueues sync jobs to refresh their data
+  #         If their data wasn't refreshed in the last 2 hours.
+  #
+  # Returns nil
+  def sync_wallet_connections
+    if gemini_connection.present? && gemini_connection.updated_at > 2.hours.ago
```
Comment (yachtcaptain23) [inline app/models/publisher.rb:168]:
```suggestion
    if gemini_connection.present? && gemini_connection.updated_at < 2.hours.ago
```

File: db/migrate/20200716170339_add_recipient_id_to_gemini_connection.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+class AddRecipientIdToGeminiConnection < ActiveRecord::Migration[6.0]
+  def change
+    add_column :gemini_connections, :recipient_id, :string, index: true, unique: true
```
Comment (yachtcaptain23) [inline db/migrate/20200716170339_add_recipient_id_to_gemini_connection.rb:3]: Did you want to set a default and allow or disallow null?

Comment (corymcdonald) [reply db/migrate/20200716170339_add_recipient_id_to_gemini_connection.rb:3]: I don't think so, if there's not a recipient_id created then this field should be null.

**Pull request merged**